README.md

```diff
@@ -48,8 +56,7 @@ Currently we support `onVrtComplete` hook, which allows you to get results of co
 One of the ways to use this data is the following:
 
 ```js
-
-(async function() {
+(async function () {
```

Comment:
if you use readme formatter - could you provide a name of it, because it might be that we use different one and we will keep producing diff in this file.

The idea is to add it to the precommit hook or github action to this repo. It should be a separate pull-request. 

Meanwhile, could you switch this off? 

These formmating changes are not necessary.